.travis.yml

```diff
@@ -2,7 +2,6 @@ sudo: false
 language: php
 
 php:
-  - 7.1
   - 7.2
```

Comment:
Shouldn't we test against 7.3 also? i.e. have Travis tests for each version of PHP that we officially support.

Reply:
I didn't add 7.3 in this PR because there's a behavior difference between 7.2 and 7.3 in some of the tests. https://github.com/bovigo/vfsStream/pull/172 adds a workaround for that.

Reply:
ah that makes sense; good call!